apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -129,6 +135,18 @@ export default function TabBuild() {
     return `${current}/${total}`
   }, [deferredBuildSetting, filteredArtIds, database])
 
+  const tabFocused = useRef(true)
```

Comment:
Why limit it to only when tab isn't focused? If user enables it, may as well play it

Reply:
could just play the sound, but send alert when its not active tab?